ui/pages/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -574,6 +588,15 @@ export default class ConfirmTransactionBase extends Component {
       </div>
     );
 
+    const hardwareLockedWarning = () => (
```

Comment:
Could we genericise/elaborate `simulationFailureWarning` to handle different types of error messages?

Reply:
This was adapted from `ui/pages/connect-account/connect-hardware/index,js#renderError`, but can be changed to use/do something else. If being used in both places, it may be worth extracting a generic component from `connect-hardware` - though this will involve deeper changes.

Reply:
The UI and polling have been extracted into the `HardwareWalletState` component which renders an `ActionableMessage` when the device is locked.